packages/app/lib/social-logins/index.ts

```diff
@@ -15,62 +13,35 @@ export const useMagicSocialAuth = () => {
     const scope = ["email"];
     const params = new URLSearchParams(config).toString();
     const redirectUri = OAUTH_REDIRECT_URI + "?" + params;
-    if (Platform.OS === "web") {
-      //@ts-ignore
-      return magic.oauth.loginWithRedirect({
-        provider: "twitter",
-        redirectURI: redirectUri,
-        scope,
-      });
-    } else {
-      return magic.oauth.loginWithPopup({
-        provider: "twitter",
-        scope,
-        redirectURI: redirectUri,
-      });
-    }
+    return magic.oauth.loginWithPopup({
+      provider: "twitter",
+      scope,
+      redirectURI: redirectUri,
+    });
   };
 
   const performMagicAuthWithGoogle = async (config?: Config) => {
     const scope = ["email"];
     const params = new URLSearchParams(config).toString();
     const redirectUri = OAUTH_REDIRECT_URI + "?" + params;
 
-    if (Platform.OS === "web") {
-      //@ts-ignore
-      return magic.oauth.loginWithRedirect({
-        provider: "google",
-        redirectURI: redirectUri,
-        scope,
-      });
-    } else {
-      return magic.oauth.loginWithPopup({
-        provider: "google",
-        scope,
-        redirectURI: redirectUri,
-      });
-    }
+    return magic.oauth.loginWithPopup({
```

Comment:
Does loginWithPopup work correctly native?

Reply:
yeah, it was originally just working on native. This PR makes it work on web.

Reply:
Test it on dev.showtime.xyz instead of preview url!